discovery-provider/src/tasks/tracks.py

```diff
@@ -62,6 +62,11 @@ def track_state_update(self, update_task, session, track_factory_txs, block_numb
             num_total_changes += len(track_events_tx)
 
     for track_id, value_obj in track_events.items():
+        # If track record object is null, it has a blacklisted metadata CID
+        if value_obj['track'] is None:
```

Comment:
just a style preference - instead of checking is None here, can we just avoid adding it i the first place?

you could just check the returned value of `parse_track_event` and avoid adding it (on line 54)


Instead of:
```
track_events[track_id]["track"] = parse_track_event
```

It could be 
```
track_parsed_value = parse_track_event
if track_parsed_value !== None:
   track_events[track_id]["track"] = track_parsed_value

Reply:
sure! we should also not be adding the bad update/new track indexing here like

```
track_parsed_value = parse_track_event
if track_parsed_value !== None:
   track_events[track_id]["track"] = track_parsed_value
   track_events[track_id]["events"].append(event_type)
```
?

Reply:
Yeah i think you're right!